helm/postgres/values.yaml

```diff
@@ -232,7 +232,7 @@ postgresVersion: 14
 # You can't set "multiBackupRepos" and any of the individual quickstarts at the
 # same time. "multiBackupRepos" will take precedence.
 #
-# Below is an example that enables one of each backup type. Note all of the
```

Comment:
I believe the original wording is correct. It is saying that the example includes all of the possible backup repo types. I guess you could add a comma after "Note" so that it is more clear.